modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -166,14 +171,16 @@ const setup = (editor: Editor): RenderInfo => {
       }
     };
 
+    const menuBarCollection = shouldHavePromotion ? [ partSpap, partMenubar ] : [ partMenubar ];
```

Comment:
"menubar" is one word, so it shouldn't be `menuBarCollection` and instead should be `menubarCollection`